nix/supervisord-cluster/topology/cardano-topology.hs

```diff
@@ -0,0 +1,247 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -p "haskellPackages.ghcWithPackages (pkgs: with pkgs; [aeson graphviz optparse-applicative split])" -i runhaskell
+
```

Comment:
Why do we have this code as she-bang Haskell script ?
Why not just a regular executable ?

Reply:
Good point -- that's no longer necessary.  It was used like this in the `cardano-ops` deployment, but now that I had to introduce the cabal thing, that's obsolete.

Good catch!